### PR TITLE
Added conmon variant build to singularityce

### PR DESCRIPTION
**Pull request by pelahi**
- also added new version entry

### CODE REVIEW

**Review by alalazo — CHANGES_REQUESTED** [review]
You can check style locally by running:
```
$ spack style [--fix]
```

File: var/spack/repos/builtin/packages/singularityce/package.py [hunk 1]
```diff
@@ -53,7 +55,7 @@ def singularity_gopath_dir(self):
     # Unpack the tarball as usual, then move the src dir into
     # its home within GOPATH.
     def do_stage(self, mirror_only=False):
-        super().do_stage(mirror_only)
+        super(SingularityBase, self).do_stage(mirror_only)
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/singularityce/package.py:58]:
```suggestion
        super().do_stage(mirror_only)
```

**Review by alalazo — CHANGES_REQUESTED** [review]
Can you please rebase, if you're still interested in the PR, or close the PR?[^1]

[^1]: When submitting from an organization we can't help shepherding PRs, since we (maintainers) are not allowed to edit the PR branch.

**Comment by pelahi** [comment]
Hi @alalazo I will be swamped for the next two weeks but will return to this as I may have further updates based on newer singularityce versions.

**Comment by alalazo** [comment]
@pelahi

**Comment by pelahi** [comment]
Hi @alalazo , I can try updating the PR but I am surprised other centres have not encountered this issue and desire this feature. I will need to update the PR to better reflect the changes in spack recipes for newer versions of spack. I will thus close this PR and open another

**Pull request closed without merging**